src/data_structures/mod.rs

```diff
@@ -1,5 +1,3 @@
+mod linked_list;
 mod queue;
-mod stack;
-
-pub use self::queue::Queue;
-pub use self::stack::Stack;
```

Comment:
If you remove these declarations the compiler cannot discover the code. This is needed in order to compile and run the code.

You can refer to https://github.com/alexfertel/rust-algorithms/blob/main/CONTRIBUTING.md#structure